components/TypeForm/PopOver.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import { useState, useEffect } from 'react';
+import { Popover } from '@typeform/embed-react';
+import { type PopoverOptions } from '@typeform/embed/types';
+
+export type PopOverProps = PopoverOptions & {
+  id: string;
+}
+
+const PopOver = ({id, ...rest}:PopOverProps) => {
+  const [isClient, setIsClient] = useState(false);
+  useEffect(() => {
+    setIsClient(true);
+  }, []);
+
+  if (!isClient) {
+    return null;
+  }
+
+  return <Popover noHeading keepSession id={id} {...rest} />;
```

Comment:
No heading text and the session is maintained if the widget gets refreshed (webpack HMR or window reload)